apisix/control/v1.lua

```diff
@@ -400,6 +400,13 @@ function _M.dump_plugin_metadata()
     return 200, metadata.value
 end
 
+function _M.plugins_reload()
```

Comment:
Missing the logic of sending events to the other worker processes.
You can refer: https://github.com/apache/apisix/blob/master/apisix/admin/init.lua#L284